boards/arduino_nano_connect/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
```

Comment:
I'm not sure if changelogs for individual board crates are really useful. Updating them will be a lot of work, and in most cases they will all contain the same entries. IIRC there was a discussion about this in the past, but I don't remember where and what the conclusion was.